VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -198,6 +202,60 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
             if currentPlayer.rate != props.rate {
                 currentPlayer.rate = props.rate
             }
+            
+            if case .supported(let active) = props.pictureInPictureMode {
+                setPictureInPictureController(toState: active)
+            }
+        }
+    }
+    
+    #if os(iOS)
+    private var _pictureInPictureController: AnyObject?
```

Comment:
Don't like this idea, a lot of check for platform and iOS, let's investigate how to make it better.